<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="3" ignored="0" total="13" passed="9" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-12T16:28:10 EDT" name="Suite" finished-at="2022-08-12T16:28:33 EDT" duration-ms="22770">
    <groups>
    </groups>
    <test started-at="2022-08-12T16:28:10 EDT" name="Landing Page in Chrome" finished-at="2022-08-12T16:28:22 EDT" duration-ms="11396">
      <class name="com.qa.app.tests.LandingPageTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.qa.app.tests.LandingPageTest@56928307]" started-at="2022-08-12T16:28:10 EDT" name="setUp" finished-at="2022-08-12T16:28:18 EDT" duration-ms="7963" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="landingPageSetUp()[pri:0, instance:com.qa.app.tests.LandingPageTest@56928307]" started-at="2022-08-12T16:28:18 EDT" name="landingPageSetUp" finished-at="2022-08-12T16:28:18 EDT" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- landingPageSetUp -->
        <test-method retried="true" signature="landingPageTittleTest()[pri:1, instance:com.qa.app.tests.LandingPageTest@56928307]" started-at="2022-08-12T16:28:18 EDT" name="landingPageTittleTest" description="Varifying Landing Page Tittle" finished-at="2022-08-12T16:28:18 EDT" duration-ms="22" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Rahul Shetty Academy123] but found [Rahul Shetty Academy]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Rahul Shetty Academy123] but found [Rahul Shetty Academy]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.app.tests.LandingPageTest.landingPageTittleTest(LandingPageTest.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- landingPageTittleTest -->
        <test-method retried="true" signature="landingPageTittleTest()[pri:1, instance:com.qa.app.tests.LandingPageTest@56928307]" started-at="2022-08-12T16:28:18 EDT" name="landingPageTittleTest" description="Varifying Landing Page Tittle" finished-at="2022-08-12T16:28:19 EDT" duration-ms="11" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Rahul Shetty Academy123] but found [Rahul Shetty Academy]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Rahul Shetty Academy123] but found [Rahul Shetty Academy]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.app.tests.LandingPageTest.landingPageTittleTest(LandingPageTest.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- landingPageTittleTest -->
        <test-method retried="true" signature="landingPageTittleTest()[pri:1, instance:com.qa.app.tests.LandingPageTest@56928307]" started-at="2022-08-12T16:28:19 EDT" name="landingPageTittleTest" description="Varifying Landing Page Tittle" finished-at="2022-08-12T16:28:19 EDT" duration-ms="11" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Rahul Shetty Academy123] but found [Rahul Shetty Academy]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Rahul Shetty Academy123] but found [Rahul Shetty Academy]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.app.tests.LandingPageTest.landingPageTittleTest(LandingPageTest.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- landingPageTittleTest -->
        <test-method signature="landingPageTittleTest()[pri:1, instance:com.qa.app.tests.LandingPageTest@56928307]" started-at="2022-08-12T16:28:19 EDT" name="landingPageTittleTest" description="Varifying Landing Page Tittle" finished-at="2022-08-12T16:28:19 EDT" duration-ms="8" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Rahul Shetty Academy123] but found [Rahul Shetty Academy]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Rahul Shetty Academy123] but found [Rahul Shetty Academy]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.app.tests.LandingPageTest.landingPageTittleTest(LandingPageTest.java:25)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- landingPageTittleTest -->
        <test-method signature="verifyHeader()[pri:2, instance:com.qa.app.tests.LandingPageTest@56928307]" started-at="2022-08-12T16:28:19 EDT" name="verifyHeader" description="Verifying the Header on Landing Page" finished-at="2022-08-12T16:28:19 EDT" duration-ms="66" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHeader -->
        <test-method signature="verifyLoginButton()[pri:3, instance:com.qa.app.tests.LandingPageTest@56928307]" started-at="2022-08-12T16:28:19 EDT" name="verifyLoginButton" description="Verifying if Login Button is clickable or not" finished-at="2022-08-12T16:28:20 EDT" duration-ms="75" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginButton -->
        <test-method signature="verifyJoinbutton()[pri:4, instance:com.qa.app.tests.LandingPageTest@56928307]" started-at="2022-08-12T16:28:20 EDT" name="verifyJoinbutton" description="Verifying is Join Button is clickable or not" finished-at="2022-08-12T16:28:20 EDT" duration-ms="71" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyJoinbutton -->
        <test-method signature="getLogin()[pri:5, instance:com.qa.app.tests.LandingPageTest@56928307]" started-at="2022-08-12T16:28:20 EDT" name="getLogin" description="Clicking on Login Button and verifying that user is on Login Page" finished-at="2022-08-12T16:28:21 EDT" duration-ms="1732" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.app.tests.LandingPageTest@56928307]" started-at="2022-08-12T16:28:21 EDT" name="tearDown" finished-at="2022-08-12T16:28:22 EDT" duration-ms="269" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.app.tests.LandingPageTest -->
    </test> <!-- Landing Page in Chrome -->
    <test started-at="2022-08-12T16:28:22 EDT" name="Tests in  Firefox" finished-at="2022-08-12T16:28:33 EDT" duration-ms="10947">
      <class name="com.qa.app.tests.LoginPageTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.qa.app.tests.LoginPageTest@6b88ca8c]" started-at="2022-08-12T16:28:22 EDT" name="setUp" finished-at="2022-08-12T16:28:27 EDT" duration-ms="5090" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="loginPageSetUp()[pri:0, instance:com.qa.app.tests.LoginPageTest@6b88ca8c]" started-at="2022-08-12T16:28:27 EDT" name="loginPageSetUp" finished-at="2022-08-12T16:28:29 EDT" duration-ms="2159" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageSetUp -->
        <test-method signature="verifyLoginPageHeader()[pri:1, instance:com.qa.app.tests.LoginPageTest@6b88ca8c]" started-at="2022-08-12T16:28:29 EDT" name="verifyLoginPageHeader" description="Verifying the Header on Login Page" finished-at="2022-08-12T16:28:29 EDT" duration-ms="47" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginPageHeader -->
        <test-method signature="verifyRememberMeLink()[pri:2, instance:com.qa.app.tests.LoginPageTest@6b88ca8c]" started-at="2022-08-12T16:28:29 EDT" name="verifyRememberMeLink" description="Verifying if Remember Me link is working or not on Login Page" finished-at="2022-08-12T16:28:29 EDT" duration-ms="57" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRememberMeLink -->
        <test-method signature="verifySignUp()[pri:3, instance:com.qa.app.tests.LoginPageTest@6b88ca8c]" started-at="2022-08-12T16:28:29 EDT" name="verifySignUp" description="Verifying if Sign Up link is working or not on Login Page" finished-at="2022-08-12T16:28:29 EDT" duration-ms="60" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySignUp -->
        <test-method signature="verifyForgotPassword()[pri:4, instance:com.qa.app.tests.LoginPageTest@6b88ca8c]" started-at="2022-08-12T16:28:29 EDT" name="verifyForgotPassword" description="Verifying if Forgot Password is working or not on Login Page" finished-at="2022-08-12T16:28:30 EDT" duration-ms="53" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyForgotPassword -->
        <test-method signature="doLogin()[pri:5, instance:com.qa.app.tests.LoginPageTest@6b88ca8c]" started-at="2022-08-12T16:28:30 EDT" name="doLogin" description="Doing Login and verifying if user is Logged in or not and is on Home Page" finished-at="2022-08-12T16:28:33 EDT" duration-ms="3283" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.app.tests.LoginPageTest@6b88ca8c]" started-at="2022-08-12T16:28:33 EDT" name="tearDown" finished-at="2022-08-12T16:28:33 EDT" duration-ms="183" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.app.tests.LoginPageTest -->
    </test> <!-- Tests in  Firefox -->
  </suite> <!-- Suite -->
</testng-results>
